README.md

```diff
@@ -63,6 +64,28 @@ err := c.Provide(&Type1{Name: "I am an thing"})
 ```
 
 
+### Provide an Maps, slices or arrays
+
+Dig is now able to support maps, slices and arrays as objects to
```

Comment:
Suggest:

"Dig also supports maps, slices, and arrays:"

No reason to say "is now able to" in the readme.